src/transaction.ts

```diff
@@ -70,12 +70,18 @@ export interface Statement {
   types?: Type | {[param: string]: Value};
 }
 
+export interface TxnRequestOptions {
+  requestTag?: string;
+  transactionTag?: string;
+}
+
 export interface ExecuteSqlRequest extends Statement, RequestOptions {
   resumeToken?: ResumeToken;
   queryMode?: spannerClient.spanner.v1.ExecuteSqlRequest.QueryMode;
   partitionToken?: Uint8Array | string;
   seqno?: number;
   queryOptions?: IQueryOptions;
+  txnRequestOptions?: Pick<TxnRequestOptions, 'requestTag'>;
```

Comment:
I worry about this in combination with #1282 
For RPC priority we add `IRequestOptions` to `ExecuteSqlRequest`, `CommitOptions` and `BatchUpdateOptions`. `IRequestOptions` will eventually (when the proto changes for RPC Prio and tagging have been merged) contain both the tagging options and the RPC priority options. If we keep these two PRs as they are now, we will introduce a very confusing situation, because:
1. A user can specify tags in `txnRequestOptions`
2. A user can specify tags in `IRequestOptions`

I don't think it will be clear to users which one they should use.

So either we need to change #1282 to also introduce a separate options interface for RPC priority, or this PR should be changed into also using `IRequestOptions` on these interfaces. My preference would be the latter to avoid cluttering up the options interfaces with two many different types.

Reply:
In https://github.com/googleapis/nodejs-spanner/pull/1282#discussion_r605310538, I was saying that I don't think we want to introduce different names for different combinations of options. We want to have a single name that mirror the proto field that it maps to. We'll try to be helpful to the user on which fields can be set by using Pick/Omit. Hope you don't mind.